packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 9.1.5
+
+-   fix(connect-iframe): not create new timestamp for proxy logs (4919dbcec3)
+-   feat(connect-common): added T2B1 frimware binaries 2.6.2 (423699284f)
+-   fix(connect-iframe): user original timestamp for proxy logs (4919dbcec3)
+-   feat(connect-common): added T2B1 frimware binaries 2.6.2 (423699284f)
+-   feat(suite): T2B1 replace Model R name by official Trezor Safe 3 name (7460372ed1)
+-   feat(connect): check that CA certificate validity from is not in the future (249ddc358a)
+-   fix(connect-common): fix bootloader version in T2B1 release config (4e698091b2)
+-   feat(connect): add production root and CA pubkeys (af907a296d)
+-   feat(connect): add authenticateDevice method (45b99c0813)
+-   feat(connect): btg, dash, dgb, nmc, vrc no support for T2B1 (0819ff6fc1)
+-   feat(suite): support t2b1 firmware installation (9ef2bf627a)
```

Comment:
also here, suite in commit message. of course this commit affected connect somehow, but mentioning suite does not bring any value for people looking for connect changes